src/systems/systems/src/pressurization/mod.rs

```diff
@@ -0,0 +1,952 @@
+use self::{cabin_pressure_controller::CabinPressureController, pressure_valve::PressureValve};
+use crate::{
+    overhead::{AutoManFaultPushButton, OnOffPushButton, ValueKnob},
+    shared::random_number,
+    simulation::{
+        Read, SimulationElement, SimulationElementVisitor, SimulatorReader, SimulatorWriter,
+        UpdateContext, Write,
+    },
+};
+use uom::si::{
+    f64::*, length::foot, pressure::hectopascal, ratio::percent, velocity::foot_per_minute,
+};
+
+mod cabin_pressure_controller;
+mod pressure_valve;
+
+pub trait PressureValveActuator {
+    fn should_open_pressure_valve(&self) -> bool;
+    fn should_close_pressure_valve(&self) -> bool;
+    fn target_valve_position(&self) -> Ratio;
+}
+
+pub struct AircraftInputsPressurization {
```

Comment:
I removed the struct `AircraftInputsPressurization` and because of a couple of the other modifications that removed the overhead panel and the outflow valve from the `CabinPressureController` type, the `#[allow(clippy::too_many_arguments)]` in not even needed now (although it might be in the future)! 😀